tests/features/common/actions/table.action.js

```diff
@@ -80,6 +85,7 @@ const action = {
     const rows = await getTableRows(driver, table)
     const arr = []
     expect(rows).not.equal(0)
+
     for (let i = rows; i >= 1; i--) {
       await hoverComponent(driver, table.tableFields[column](i)['label'])
       const text = await getElementText(
```

Comment:
move variable declaration to the top of loop, add empty line after variable declaration